### ISSUE TITLE
Implicit annotation in dependency injection not minifiable

### ISSUE THREAD

**Issue by rahelest** [neoziro/angular-clickout#1]
Hello!
Thank you for this nice directive!
However, there seems to be an issue with the minified version.

This part of code: 
```javascript
    .directive("clickOut",function(a,c)
```

Causes 
>    "Unknown provider: aProvider <- a <- clickOutDirective"

Example fix with inline array annotation ([Angular Doc](https://docs.angularjs.org/guide/di)):
```javascript
        .directive("clickOut",['$window', '$parse', function(a,c) {..... }])
```

**Comment by piyushchauhan2011** [comment]
:+1: Please correct it.

**Comment by neoziro** [comment]
Fixed in #2, released in https://github.com/neoziro/angular-clickout/releases/tag/v1.0.2
